src/util/schnorr.rs

```diff
@@ -192,9 +192,21 @@ impl TweakedKeyPair {
 
     /// Returns the underlying key pair
     #[inline]
+    #[deprecated(since = "0.29.0", note = "use to_inner instead")]
     pub fn into_inner(self) -> crate::KeyPair {
         self.0
     }
+
+    /// Returns the underlying key pair.
+    #[inline]
+    pub fn to_inner(self) -> crate::KeyPair {
+        self.0
+    }
+
+    /// Returns a reference to underlying key pair.
+    pub fn as_inner(&self) -> &crate::KeyPair {
```

Comment:
In 73c5f628b60ec55a5df221bbcf2795eb35311041:

Because both the source and target types are `Copy` I think this method is unnecessary.

Reply:
Sweet, I'll put a patch at the front of this PR that removes them.